src/server/game/Spells/SpellScript.cpp

```diff
@@ -32,6 +32,23 @@ bool _SpellScript::_Validate(SpellInfo const* entry)
     return true;
 }
 
+bool _SpellScript::_ValidateSpellInfo(uint32 const* begin, uint32 const* end)
```

Comment:
Already have that one with better arguments.
`_SpellScript::ValidateSpellInfo(std::vector<uint32> const& spellIds)`